### PR TITLE
docs: design document for kubectl config directory

### PR DESCRIPTION
**Pull request by dwmkerr**

### CODE REVIEW

File: contributors/design-proposals/cli/kubectl-config-directory.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+The following use-cases have been considered:
+
+#### Rapid Access to Newly Created Clusters
+
+Consider the following project:
+
+[terraform-aws-openshift](https://github.com/dwmkerr/terraform-aws-openshift)
+
+This project creates a Kubernetes Cluster. One the cluster has been created, it is convenient to offer the user a way to access it. At the moment, the only options are:
```
Comment (dperique) [inline contributors/design-proposals/cli/kubectl-config-directory.md]: s/One/Once/

**Comment by dwmkerr** [comment]
/remove-lifecycle rotten
requesting review from anyone! cc @fejta

**Comment by mrbobbytables** [comment]
@dwmkerr I'm sorry it's taken so long for someone to get back on this =/, but the design-proposal process was superseded by KEPs (Kubernetes Enhancement Proposals) in late 2018.

Reference info:
- https://github.com/kubernetes/enhancements/issues/617
- https://github.com/kubernetes/enhancements/pull/638
- https://groups.google.com/d/topic/kubernetes-dev/LIkZoIqCT20/discussion
- https://github.com/kubernetes/enhancements/blob/master/keps/0001a-meta-kep-implementation.md

If you're still interested in moving forward with this, I'd encourage reaching out on the [sig-cli mailing list](https://github.com/kubernetes/community/tree/master/sig-cli#contact) or in their slack. From there it could be moved to a KEP 👍 


/cc @justaugustus

**Comment by gkarthiks** [comment]
hi @dwmkerr Since we have a KEP process, can we please merge this soon or close this and create a KEP?

**Comment by justaugustus** [comment]
Closing this as it should be a KEP as @mrbobbytables mentioned.
Please discuss with SIG CLI --> https://github.com/kubernetes/community/tree/master/sig-cli#contact

/close

**Pull request closed without merging**